conf/application.conf

```diff
@@ -113,5 +113,5 @@ tracking-consent-frontend {
 }
 
 features {
-  welsh-translation: true
+  welsh-translation: false
```

Comment:
Set this to false to remove banner and make page look like prototype (This can be changed if welsh translation ever is needed in future but I don't think it will be)